chapters/cpp_for_opencl.md

```diff
@@ -12,8 +12,103 @@ C++ for OpenCL is supported by [Clang](https://clang.llvm.org/docs/UsersManual.h
 <br> <br>
 </p>
 
-OpenCL C code is valid and fully compatible with C++ for OpenCL. This enables developers to use a consistent front-end compiler as they incrementally transition from OpenCL C to use C++ features for their applications.
-
-C++ for OpenCL generates SPIR-V 1.0 plus SPIR-V 1.2 where necessary. Experimental support was added in [Clang 9](https://clang.llvm.org/docs/UsersManual.html#cxx-for-opencl) with bug fixes and improvements in [Clang 10](https://releases.llvm.org/10.0.0/tools/clang/docs/ReleaseNotes.html#opencl-kernel-language-changes-in-clang).
+Experimental support for C++ for OpenCL was added in [Clang 9](https://releases.llvm.org/9.0.0/tools/clang/docs/LanguageExtensions.html#opencl-features) with bug fixes and improvements in [Clang 10](https://releases.llvm.org/10.0.0/tools/clang/docs/ReleaseNotes.html#opencl-kernel-language-changes-in-clang).
 
 You can check out C++ for OpenCL in [Compiler Explorer](https://godbolt.org/z/NGZw9U).
+
+## Documentation
+
+The language documentation can be found in releases of [OpenCL-Docs](https://github.com/KhronosGroup/OpenCL-Docs/releases) with [the first official](https://github.com/KhronosGroup/OpenCL-Docs/releases/tag/cxxforopencl-v1.0-r1) version 1.0 (see also [the latest WIP version](OpenCL](https://github.com/KhronosGroup/Khronosdotorg/blob/master/api/opencl/assets/CXX_for_OpenCL.pdf).
```

Comment:
There's a minor Markdown bug here that's causing the link to be broken.  I think the way to fix it is to remove `(OpenCL]`, but please review to be sure.

Reply:
Fixed! Thanks!